plugins/remark-custom-blocks.js

```diff
@@ -18,6 +18,8 @@ const extractText = (nodes) => {
 const plugin = (options) => {
     const transformer = (tree, file) => {
 
+        let total_steps = 0; // Keep track of the total number of steps
```

Comment:
Added because I need to keep track of **total** number of steps because you might have multiple of these `<VerticalStepper>` components and my hacky DOM manipulation finds elements on `step-X` so I was getting interference between components.